crv/crvQuality.cc

```diff
@@ -142,39 +142,39 @@ static double getTetPartialJacobianDet(apf::NewArray<apf::Vector3>& nodes,
  *
  */
 static double Nijk(apf::NewArray<apf::Vector3>& nodes,
-    int d, int I, int J)
+    int d, int II, int J)
```

Comment:
What was the problem here with `I`?

Reply:
When we compile with C-complex instead of C++-complex the "complex.h" header declares an `I` macro for the imaginary part of complex numbers in the global namespace... which is a hasstle. I could also `#undef I` in the `apfComplexType.h` header if that would be preferred, but that might have other issues?

Reply:
Ahhh.  Interesting.  The `#undef` seems messier than just avoiding `I`.